plugins/speculation-rules/readme.txt

```diff
@@ -88,6 +88,16 @@ add_filter(
 
 As mentioned above, adding the `no-prerender` CSS class to a link will prevent it from being prerendered (but not prefetched). Additionally, links with `rel=nofollow` will neither be prefetched nor prerendered because some plugins add this to non-idempotent links (e.g. add to cart); such links ideally should rather be buttons which trigger a POST request or at least they should use `wp_nonce_url()`.
 
+= How will this impact analytics and personalization? =
+
+Prerendering can affect analytics and personalization.
+
+For client-side JavaScript, is recommended to delay these until the page clicks and some solutions (like Google Analytics) already do this automatically for prerender. See [Impact on Analytics](https://developer.chrome.com/docs/web-platform/prerender-pages#impact-on-analytics). Additionally, cross-origin iframes are not loaded until activation which can further avoid issues here.
```

Comment:
What is "until the page clicks"? Is that the most suitable term here? I would assume there's an easier-to-understand way to indicate that the page has actually become active / that the user is actually visiting the page.

Reply:
Fair point. The term we usually use is "when the prerender is activated"/"activation" since this matches the [`activationStart ` time in the Performance Navigation Timing API](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigationTiming/activationStart)

```suggestion
For client-side JavaScript, it is recommended to delay these until the prerender is activated (for example but clicking on the like) and some solutions (like Google Analytics) already do this automatically for prerender. See [Impact on Analytics](https://developer.chrome.com/docs/web-platform/prerender-pages#impact-on-analytics). Additionally, cross-origin iframes are not loaded until activation which can further avoid issues here.
```